firewall/Dockerfile

```diff
@@ -20,7 +23,8 @@ RUN apt-get update --quiet \
     fever \
     tcpdump \
  && apt-get remove --yes iptables \
- && apt --yes autoremove
+ && apt --yes autoremove \
+ && apt install --yes iptables
```

Comment:
just dont remote iptables, add it to the list of packages to be installed and remove the apt-get remove iptables step